assets/js/modules/analytics/components/settings/SettingsEdit.test.js

```diff
@@ -32,6 +32,10 @@ describe( 'SettingsEdit', () => {
 			/tagmanager\/data\/settings/,
 			{ body: {}, status: 200 },
 		);
+		fetchMock.getOnce(
+			/^\/google-site-kit\/v1\/modules\/analytics-4\/data\/properties/,
+			{ body: [], status: 200 },
```

Comment:
The `status` is already `200` by default, so we can omit it:
```suggestion
			{ body: [] },
```